sdk/storage/azure-storage-blob/src/main/java/com/azure/storage/blob/specialized/BlobOutputStream.java

```diff
@@ -40,7 +40,7 @@ static BlobOutputStream appendBlobOutputStream(final AppendBlobAsyncClient clien
         return new AppendBlobOutputStream(client, appendBlobRequestConditions);
     }
 
-    static BlobOutputStream blockBlobOutputStream(final BlockBlobAsyncClient client,
+    public static BlobOutputStream blockBlobOutputStream(final BlobAsyncClient client,
```

Comment:
should I rename this to blobOutputStream or is that confusing?

Reply:
Leave the name as-is, this stream is meant to solely interact with Block Blobs, don't want to conflate the overarching Blob term to mean Block Blob (though that is the primary use case).